server/routes/standardRouter.ts

```diff
@@ -1,22 +1,24 @@
 import { Router } from 'express'
+import csurf from 'csurf'
 
-// const testMode = process.env.NODE_ENV === 'test'
+const testMode = process.env.NODE_ENV === 'test'
 
 export function standardRouter(): Router {
-  //
-  // // CSRF protection
-  // if (!testMode) {
-  //   router.use(csurf())
-  // }
-  //
-  // router.use((req, res, next) => {
-  //   if (typeof req.csrfToken === 'function') {
-  //     res.locals.csrfToken = req.csrfToken()
-  //   }
-  //   next()
-  // })
+  const router = Router({ mergeParams: true })
 
-  return Router({ mergeParams: true })
+  // CSRF protection
+  if (!testMode) {
```

Comment:
since the notification router was the last one to be moved, this code can now be used for the csrf protection